### PR TITLE
Urgent spammers

### PR DESCRIPTION
**Pull request by JasonD94**
13 new spammers I found, either new ones from today or from searching on the keyword "URGENT" in my inbox. Will try looking for more later.

### CODE REVIEW

File: list.txt [hunk 1]
```diff
@@ -20,13 +22,17 @@ avanceservices.us
...
 bytesys.net
 canbayinc.com
 cbsinfosys.com
 codeforce.com
 compnova.com
 compugain.com
+concept-inc.com
 consultnet.com
+cybercoders.com
+mail.cybercoders.com
```
Comment (johnceloria) [inline list.txt:35]: Does your filter not block the entire domain? I know the gmail filter does which makes this redundant.

Comment (jceloria) [inline list.txt]: Does your filter not block the entire domain? This is redundant in the gmail filter.

Comment (JasonD94) [reply list.txt]: Ah yes you're correct. I'll remove that second one.

Comment (JasonD94) [reply list.txt]: Just removed that extra email.

**Pull request merged**